src/components/expansion-panel/expansion-panel.ts

```diff
@@ -109,7 +109,7 @@ export default class IgcExpansionPanelComponent extends EventEmitterMixin<
    * @private
    * Opens the panel.
    */
-  private openWithEvent() {
+  public openWithEvent() {
```

Comment:
Not a fan of this change at all.

If you need to emit events, you can either:
1. Dispatch the event from the accordion using `dispatchEvent` with the appropriate event object.
2. Emit the events from the accordion itself.